tests/examples/source/devfiles/nodejs/devfile-autobuild-deploybydefault.yaml

```diff
@@ -37,7 +37,7 @@ commands:
 
   - composite:
       commands:
-        - image-autobuild-true-and-referenced
```

Comment:
Perhaps what you had makes sense while reading it. Can you revert this change?

Reply:
Indeed. See [`4bdcec6` (#6720)](https://github.com/redhat-developer/odo/pull/6720/commits/4bdcec60089362915b518c41b85929532faddc53)